Documentation/ApiOverview/CommandControllers/_Tutorial/_InitializeRequest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3docs\Examples\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use TYPO3\CMS\Core\Core\Bootstrap;
+use TYPO3\CMS\Core\Core\SystemEnvironmentBuilder;
+use TYPO3\CMS\Core\Http\ServerRequest;
+use TYPO3\CMS\Core\Mail\FluidEmail;
+use TYPO3\CMS\Core\Mail\MailerInterface;
+use TYPO3\CMS\Core\Site\SiteFinder;
+use TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface;
+
+#[AsCommand(
+    name: 'examples:dosomething',
+    description: 'A command that does nothing and always succeeds.',
+    aliases: ['examples:dosomethingalias'],
+)]
+class DoSomethingCommand extends Command
+{
+    public function __construct(
+        private readonly ConfigurationManagerInterface $configurationManager,
+        private readonly SiteFinder $siteFinder,
+        private readonly MailerInterface $mailer,
+        private readonly ExampleRepository $exampleRepository,
```

Comment:
If this is a extbase repository, this should fail in v13+. Request set in ConfigurationManager(Interface) already reaquired for that. Schnitzler worked
on a change to mitigate this, not sure if that has been merged or not.

Reply:
I left out the extbase examples for now and only mentioned how to send a mail with a link